server/heroku/logs.go

```diff
@@ -20,13 +21,14 @@ type PostLogsForm struct {
 func (h *PostLogs) ServeHTTPContext(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
 	a, err := findApp(ctx, h)
 	if err != nil {
-		return err
+		return fmt.Errorf("error finding app: %v", err)
```

Comment:
But, we could change the `findApp` method to handle gorm.RecordNotFound, and return a more descriptive `ErrorResource` from there. e.g:

```go
func findApp(ctx context.Context, e interface {
        AppsFind(empire.AppsQuery) (*empire.App, error)
}) (*empire.App, error) {
        vars := httpx.Vars(ctx)
        name := vars["app"]
        reporter.AddContext(ctx, "app", name)

        a, err := e.AppsFind(empire.AppsQuery{Name: &name})
        if err == gorm.RecordNotFound {
                return errNotFound(fmt.Sprintf("could not find app with name %s", name))
        } else if err != nil {
                return fmt.Errorf("error finding app: %v")
        }
        return a, nil
}
```